build/template-install-keyvault-secrets.yaml

```diff
@@ -8,12 +8,12 @@ steps:
   inputs:
     azureSubscription: 'AuthSdkResourceManager' # string. Workload identity service connection to use managed identity authentication
     KeyVaultName: 'buildautomation' # string. Required. The name of the Key Vault containing the secrets.
-    #setting secrets filter to fetch only AzureADIdentityDivisionTestAgentCert cert from the vault
```

Comment:
Why do we need to change the cert?

Reply:
because the `LabVaultAccessCert` is maintained by the lab

Reply:
to avoid confusion, I now directly refer to the lab cert from the lab vault